### ISSUE TITLE
Enhance `lit/no-invalid-escape-sequences` to have a fix 

### ISSUE THREAD

**Issue by stramel** [43081j/eslint-plugin-lit#106]
We should be able to inject an extra escape sequence as a fix for the rule.

```js
html`foo \0123 bar`
```

to 

```js
html`foo \\0123 bar`
```
